src/pylogctx/celery.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import
+
+from celery.app import current_task
+from celery.app.task import Task
+
+from .core import context
+
+
+class LoggingTask(Task):
+    def __call__(self, *args, **kwargs):
+        task = current_task()
+        context.update(
+            celeryTask=task.name,
+            celeryTaskId=task.request.id,
```

Comment:
That's a good point to discuss. I agree with you.

The purpose of these fields is to show in graylog, kibana, loggly, logmatic, etc. So this would be nice to be consistent in these interfaces.